src/workloads/tpch/normalized/1/Q1.yml

```diff
@@ -0,0 +1,28 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-query"
+Description: |
+  Run TPC-H query 1 against the normalized schema for scale 1.
+
+Clients:
+  Default:
+    QueryOptions:
+      socketTimeoutMS: -1
+
+Actors:
+- Name: TPCHNormalizedQuery1
+  Type: RunCommand
+  Database: tpch
+  Phases:
+  - LoadConfig:
+      Path: ../../../../phases/tpch/normalized/Q1.yml
+      Key: TPCHNormalizedQuery1
+      Parameters:
+        Query1Delta: -90
+
+AutoRun:
```

Comment:
Given that these workloads all run in one DSI test_control, you might want to remove the `AutoRun` section here. I'd expect otherwise the autorun scheduler would schedule all these as separate workload tasks.